### ISSUE TITLE
Add support for piping

### ISSUE THREAD

**Issue by TKlement** [Parakoopa/riptide-cli#1]
It would be a nice feature to use pipes in the terminal.
For example getting informations with magerun and pipe the output into a TXT or CSV file.

**Comment by L3P3** [comment]
Yes, also for db-import piping is much smarter. But you can usually also just use /dev/stdout or /dev/stdin instead of a file to get the same effect.
